test/hello_world.test.js

```diff
@@ -1,120 +0,0 @@
-var test = require('tape');
```

Comment:
Per chat with Dane, this will be a separate subdirectory. I plan to implement this (as well as more example) in future branches that will be merged into master...in due time.